mappings/net/minecraft/world/RegistryProvider.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5423 net/minecraft/world/RegistryProvider
```

Comment:
Hmm, what's basis for putting this in world? should it be in `net/minecraft/util/registry`?

Reply:
I put it in the world package solely because it extends interfaces that are related to world interfaces. Maybe the naming is not correct? The implementing classes are all world-related classes. Considering that and the overriden methods, what do you think of `RegistryfulWorld` instead? This is to show that the world has registry manager.

P/S: I have updated my 'analysis' in my first comment above.